frontend/packages/helm-plugin/integration-tests/features/helm/helm-navigation.feature

```diff
@@ -0,0 +1,126 @@
+@helm
+Feature: Navigations on Helm Chart
+              As a user, I want to navigate to different pages related to Helm Charts
+
+        Background:
+              Given user has created or selected namespace "aut-helm"
+
+        @smoke
```

Comment:
Can you please add two lines between two scenarios, Background and next scenario.

Reply:
updated